### PR TITLE
comment status change migration

### PR DESCRIPTION
**Pull request by Gauravano**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by Gauravano** [comment]
@jywarren could you please help me here. Thanks!

**Comment by jywarren** [comment]
I'll try to take a look later today - in the meantime, maybe @publiclab/reviewers could offer some assistance?

File: db/schema.rb.example [hunk 1]
```diff
@@ -11,446 +10,418 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180618184048) do
+ActiveRecord::Schema.define(version: 2018_07_03_164027) do
 
-  create_table "answer_selections", force: true do |t|
+  create_table "answer_selections", force: :cascade do |t|
```
Comment (jywarren) [inline db/schema.rb.example]: Hmm, maybe @Souravirus has thoughts on this and other edits to the schema example file?

File: db/migrate/20180703164027_change_status_of_comments.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class ChangeStatusOfComments < ActiveRecord::Migration[5.1]
+  def up
+    Comment.all.each do |comment|
+      comment.status = 1
+      comment.save({})
```
Comment (jywarren) [inline db/migrate/20180703164027_change_status_of_comments.rb]: is the `{}` required? Were you able to test this out locally? 

In general it looks good to me!

**Comment by jywarren** [comment]
Error is: `Mysql2::Error: BLOB/TEXT column 'params' used in key specification without a key length`

Ah, i think the changes to the schema file are overly broad -- and does this migration affect the schema? I think it may not except for the timestamp, right?

File: db/migrate/20180703164027_change_status_of_comments.rb (hunk 2 above)
Comment (Gauravano) [reply db/migrate/20180703164027_change_status_of_comments.rb]: yes, migration worked locally.

**Comment by Gauravano** [comment]
yes, it shouldn't affect schema, but on running `rake db:migrate` on my system. I am getting all these changes. Should I just change the timestamp?

File: db/schema.rb.example (hunk 1 above)
Comment (Souravirus) [reply db/schema.rb.example]: Yeah, this change looks good to me

File: db/migrate/20180703164027_change_status_of_comments.rb (hunk 2 above)
Comment (siaw23) [reply db/migrate/20180703164027_change_status_of_comments.rb]: @Gauravano i'm curious, what's the purpose of this `comment.status = 1`

Comment (Gauravano) [reply db/migrate/20180703164027_change_status_of_comments.rb]: Hi @siaw23, actually to make spam moderation system for comments, comments would come with 2 status values `status = 1` for valid and `status = 0` for spam.

**Comment by Gauravano** [comment]
@jywarren I opened a new PR for migration at #3006 . Please review. Thanks!

**Pull request closed without merging**